src/menu.js

```diff
@@ -0,0 +1,124 @@
+
+    title.innerText = 'Menu';
+    foodTitle.innerText = 'Food';
+    drinksTitle.innerText = 'Drinks';
+
+    articleDrinks.appendChild(drinksTitle);
+    articleFoods.appendChild(foodTitle);
+
+    drinks.forEach(drink => {
+
+        const name = document.createElement('p');
+        const price = document.createElement('p');
+        const description = document. createElement('p');
+        const product = document.createElement('div');
+
+        name.innerText = drink;
+        price.innerText = '$' + Math.floor(Math.random() * 30).toString();
+        description.innerText = 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio, non.';
+
+        name.classList.add('name');
+        price.classList.add('price');
+        description.classList.add('description');
+
+        product.appendChild(name);
+        product.appendChild(price);
+        product.appendChild(description);
+        articleDrinks.appendChild(product);
+
+    });
+
+    foods.forEach(food => {
+
+        const name = document.createElement('p');
+        const price = document.createElement('p');
+        const description = document. createElement('p');
+        const product = document.createElement('div');
+
+        name.innerText = food;
+        price.innerText = '$' + Math.floor(Math.random() * 30).toString();
+        description.innerText = 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio, non.';
+
+        name.classList.add('name');
+        price.classList.add('price');
+        description.classList.add('description');
+
+        product.appendChild(name);
+        product.appendChild(price);
+        product.appendChild(description);
+        articleFoods.appendChild(product);
+
+    });
+
+    menu.classList.add('menu');
+    menu.classList.add('container');
+    foodTitle.classList.add('menu-title');
+    drinksTitle.classList.add('menu-title');
+    externalBorder.classList.add('external-border');
+    articleContainer.classList.add('external-border');
+    articleContainer.classList.add('external-border');
+    articleContainer.classList.add('article-container');
+
+    articleContainer.appendChild(articleDrinks);
+    articleContainer.appendChild(articleFoods);
+    externalBorder.appendChild(articleContainer);
+    menu.appendChild(title);
+    menu.appendChild(externalBorder);
+
+    return menu;
+
+};
+
+// method for create a basic image gallery
+const gallery = () => {
+
+    const gallery = document.createElement('div');
+    const images = [G1, G2, G3, G4, G5, G6, G7, G8];
+
+    images.forEach(element => {
+
+        const imgDiv = document.createElement('div');
+        const img = document.createElement('img');
+
+        img.src = element;
+
+        imgDiv.classList.add('img-container');
+
+        imgDiv.appendChild(img);
+        gallery.appendChild(imgDiv);
+
+    });
+
+    gallery.classList.add('gallery');
+
+    return gallery;
+
+};
+
+export {
+
+    menu,
+    gallery,
```

Comment:
As seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/tab_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) you should have had these two components wrapped in a single `div`.
Ideally, I would not have requested the change but since you left out the scripts in package.json which is a requirement, this should be a simple fix.